homeassistant/components/remote/__init__.py

```diff
@@ -38,8 +41,12 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SEND_COMMANDS = 'send_commands'
```

Comment:
You can setup the send_command schema validation in such a way that it is backwards compatible but still takes in multiple commands.

I prefer to keep it in 1 function.